hack/deploy.sh

```diff
@@ -10,9 +10,9 @@ DEFAULT_OPERAND_IMAGE="quay.io/cincinnati/cincinnati:latest"
 RELATED_OPERATOR_IMAGE="${RELATED_IMAGE_OPERATOR:-${DEFAULT_OPERATOR_IMAGE}}"
 RELATED_OPERAND_IMAGE="${RELATED_IMAGE_OPERAND:-${DEFAULT_OPERAND_IMAGE}}"
 
-if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
-	RELATED_OPERATOR_IMAGE="registry.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:updateservice-operator"
-	GRAPH_DATA_IMAGE="registry.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:updateservice-graph-data-container"
+if [ -n "$OPENSHIFT_CI" ]; then
+	RELATED_OPERATOR_IMAGE="registry.ci.openshift.org/${NAMESPACE}/stable:updateservice-operator"
```

Comment:
Where does it get the namespace from now? Below we set `NAMESPACE` to our default where we want to create our resources but where does this CI stuff reside? I looked in the CI doc but didn't see anything related or I missed it. If it just in the same namespace as our op we can move the def up.

Reply:
Right, we don't need externally set `NAMESPACE` (all image pullspecs are being passed as env vars)